react/features/base/tracks/functions.any.ts

```diff
@@ -52,7 +53,7 @@ export function isParticipantMediaMuted(participant: IParticipant, mediaType: Me
  * @param {IReduxState} state - Global state.
  * @returns {boolean} - Is audio muted for the participant.
  */
-export function isParticipantAudioMuted(participant: IParticipant, state: IReduxState) {
+export function isParticipantAudioMuted(participant: IParticipant | undefined, state: IReduxState) {
```

Comment:
can `participant` be undefined? optional params go to the end, if we got undefined maybe there s some bug?

Reply:
I'll change it